index.html

```diff
@@ -0,0 +1,67 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <title>portfolio</title>
+    <link
+      href="https://fonts.googleapis.com/css2?family=Roboto&display=swap"
+      rel="stylesheet"
+    />
+    <link rel="stylesheet" href="style.css" />
+  </head>
+  <body>
+    <header>
+      <nav class="nav-container">
+        <div class="time-network">
+          <h4>9:41</h4>
+          <ul class="network-only">
+            <li class="network-only-item">
+              <img src="image/CellularConnection.png" alt="" />
+            </li>
+            <li class="network-only-item">
+              <img src="image/Wifi.svg" alt="" />
+            </li>
+            <li class="network-only-item">
+              <img src="image/Capacity.svg" alt="" />
+            </li>
+          </ul>
+        </div>
+        <div class="name-and-burger">
+          <h2>Yohannes Dag</h2>
+          <img src="image/Union.svg" class="burger" alt="none" />
+        </div>
+      </nav>
+      <div class="home-container">
+        <h1 class="greeting">
+          Hey There .<br />
+          I'm Alex
+        </h1>
+        <h2 class="job">I’am a Software Developer</h2>
+        <p class="about-me">
+          I can help you build a product , feature or website Look through some
+          of my work and experience! If you like what you see and have a project
+          you need coded, don’t hestiate to contact me.
+        </p>
+        <ul class="social-link-list">
+          <li class="social-link-item">
+            <a href="#"><img src="image/Vector.svg" alt="" /></a>
+          </li>
+          <li class="social-link-item">
+            <a href="#"><img src="image/Vector5.svg" alt="" /></a>
+          </li>
+          <li class="social-link-item">
+            <a href="#"><img src="image/Vector4.png" alt="" /></a>
+          </li>
+          <li class="social-link-item">
+            <a href="#"><img src="image/vector6.svg" alt="" /></a>
+          </li>
+          <li class="social-link-item">
+            <a href="#"><img src="image/Vector7.svg" alt="" /></a>
+          </li>
+        </ul>
+      </div>
```

Comment:
- [ ] Kindly use a `section` tag instead of the `div` tag here to make your page semantic. A semantic element clearly describes its meaning to both the browser and the developer. :writing_hand: